data/json/items/gunmod/laser_gunmods.json

```diff
@@ -0,0 +1,59 @@
+[
+  {
+    "id": "beam_scatterer",
+    "type": "GUNMOD",
+    "name": "beam scatterer",
+    "description": "A set of optics made to fit on laser weapons, which will diffract the laser beam into several lower powered beams.  This increases point-blank damage, but greatly reduces range.",
+    "weight": 380,
+    "volume": 1,
+    "price": 84000,
+    "material": [ "steel" ],
+    "symbol": ":",
+    "color": "light_gray",
+    "location": "lens",
+    "mod_targets": [ "rifle", "pistol" ],
+    "min_skills": [ [ "weapon", 4 ] ],
+    "damage_modifier": 10,
+    "range_modifier": -25,    
+    "ammo_effects" : [ "SHOT" ]
+  },
+  {
+    "id": "electrolaser_conversion",
+    "type": "GUNMOD",
+    "name": "electrolaser conversion",
+    "description": "A set of high-tech electronics and optics.  These converts a laser pistol into a less-lethal electrolaser capable of stunning targets, at the cost of a decreased damage output and increased power consumption.",
+    "weight": 380,
+    "volume": 4,
+    "price": 84000,
+    "material": [ "steel" ],
+    "symbol": ":",
+    "color": "light_gray",
+    "location": "lens",
+    "mod_targets": [ "pistol" ],
+    "min_skills": [ [ "weapon", 4 ], [ "electronics", 5 ] ],
+    "damage_modifier": -8,
+    "range_modifier": -25,    
+    "ups_charges": 10,
+    "ammo_effects" : [ "BEANBAG" ]
+  },
+  {
+    "id": "high_density_capacitor",
+    "type": "GUNMOD",
+    "name": "high density capacitor",
+    "description": "A capacitor with a higher energy density increases range, damage, and the likelihood of setting the target on fire;  at the cost of a markedly increased power consumption.",
+    "weight": 380,
+    "volume": 1,
+    "integral_volume": 0,
+    "price": 84000,
+    "material": [ "steel" ],
+    "symbol": ":",
+    "color": "light_gray",
+    "location": "emmitter",
+    "mod_targets": [ "pistol", "rifle" ],
+    "min_skills": [ [ "weapon", 4 ], [ "electronics", 5 ] ],
+    "damage_modifier": 10,
+    "range_modifier": 5,    
+    "ups_charges": 20,
+    "ammo_effects" : [ "FLARE" ]
```

Comment:
The effect was on purpose, but yes it doesnt make any sense from a realism point of view. I will remove it

Laser weapons already have incendiary, just going to leave the mod at extra damage